examples/SwissProt/SwissProt_example.md

```diff
@@ -0,0 +1,156 @@
+    ## 1 3D-structure, Acetylation, Activator, Alternative initiation, Alternative splicing, Cell cycle, Chromosome, Complete proteome, Cytoplasm, Direct protein sequencing, Disease mutation, DNA damage, DNA recombination, DNA repair, DNA-binding, Fatty acid biosynthesis, Fatty acid metabolism, Isopeptide bond, Lipid biosynthesis, Lipid metabolism, Metal-binding, Nucleus, Phosphoprotein, Polymorphism, Reference proteome, Repeat, Transcription, Transcription regulation, Transferase, Tumor suppressor, Ubl conjugation, Ubl conjugation pathway, Zinc, Zinc-finger
+    ##   sequence_update                  _id              organism data_class
+    ## 1      1995-02-01 AVteL4RIC0a5BPxHLn8w Homo sapiens (Human).   Reviewed
+
+    dim(swissprot)
+
+    ## [1]  1 26
+
+SwissProt provides extensive protein annotation information. SolveBio
+enables easy access and usability of this key information by extracting
+it into its own 'Features' dataset, where each feature is keyed by
+swissprot\_id. For our example, let's extract the "DOMAIN" features for
+"BRCA1". We need to filter the 'Features' dataset by BRCA1's swissprot
+ID and by the feature category 'DOMAIN'.
+
+    # define the filter
+    swissprot_feature_filter = list(list('swissprot_id', paste(swissprot$swissprot_id)), list('feature', paste('DOMAIN')))
+    # query the dataset with the filter
+    swissprot_features = Dataset.query('SwissProt/1.0.0-2017-02/Features', filters=swissprot_feature_filter, paginate=TRUE)
+    colnames(swissprot_features)
+
+    ##  [1] "_commit"             "swissprot_id"        "stop"               
+    ##  [4] "feature"             "start"               "stop_value"         
+    ##  [7] "start_value"         "_id"                 "feature_source"     
+    ## [10] "feature_description"
+
+    swissprot_features
+
+    ##              _commit swissprot_id stop feature start stop_value
+    ## 1 431823644774995769       P38398 1855  DOMAIN  1756       1855
+    ## 2 431823644774995769       P38398 1736  DOMAIN  1642       1736
+    ##   start_value                  _id feature_source
+    ## 1        1756 AVteMprXsOzr6UjlvC9O               
+    ## 2        1642 AVteMprXsOzr6UjlvC9P               
+    ##                                feature_description
+    ## 1 BRCT 2. {ECO:0000255|PROSITE- ProRule:PRU00033}.
+    ## 2 BRCT 1. {ECO:0000255|PROSITE- ProRule:PRU00033}.
+
+    dim(swissprot_features)
+
+    ## [1]  2 10
+
+From here, let's extract the actual amino acid sequences for the
+domains:
+
+    # add the sequence from SwissProt to the Features data frame
+    swissprot_features <- merge(swissprot_features, swissprot[, c("swissprot_id", "sequence")], by="swissprot_id")
+    swissprot_features <- as.data.table(swissprot_features)
+    # add a column "domain_sequence" and extract from the whole sequence
+    swissprot_features[,domain_sequence:=substr(sequence,start_value,stop_value)]
```

Comment:
I don't get any output for this command

```
> swissprot_features[,domain_sequence:=substr(sequence,start_value,stop_value)]
> 
```

how do I get the below output to show?